packages/common/core-interfaces/src/provider.ts

```diff
@@ -22,7 +22,6 @@
  * This pattern enables discovery, and delegation in a standard way which is central
  * to FluidObject pattern
  *
- * @internal
```

Comment:
This was causing build break. So removed it as it has been exported from the index.ts

Reply:
Do you have an example of the error it was causing? Removing it shouldn't have any functional implications but it serves some purpose being there, so I want to understand how would it cause issues with the build.